static/explain.js

```diff
@@ -1,4 +1,6 @@
-var base_url = $('#base-url').data('val');
+//used in base.html
+var base_url = 'http://127.0.0.1:8097/';
```

Comment:
Assignment to base url must not be hardcoded, it must be parameterised.

Reply:
Ok thank you, since I think I should have fixed the parameterized aspect. Please just let me know if for some reason It doesn't work. Thank you.